### ISSUE TITLE
Won't open on mac OS X 10.14.4

### ISSUE THREAD

**Issue by 265lutab** [colmap/colmap#617]
**Describe the bug**
After downloading either the release version or the dev version and extracting the app I tried opening it either with right click or double click and nothing happens. No crash notification or anything it just doesn't open. I'm running Mojave 10.14.4. I also tried deleting and downloading it again.

**To Reproduce**
Open the app.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS Mojave 10.14.4
 - Browser downloaded with Firefox

**Smartphone (please complete the following information):**

**Additional context**

**Comment by 265lutab** [comment]
I had an old version of colmap.app in the same folder so the new one ended up being called colmap 2.app. When I deleted the old one and renamed the new one it worked.

**Comment by ahojnnes** [comment]
Yes, I also noticed that in the past. However, I am not an expert in Mac app bundling. If anyone knows a workaround for this, then a fix for this would be greatly appreciated.

**Comment by ahojnnes** [comment]
Closing as this was a user error on macOS Mojave (duplicate app naming issue). The reporter resolved it by renaming, and this macOS version is no longer supported.
